### PR TITLE
(AUTOMATIC) opensource update

### PR DESCRIPTION
**Pull request by sebsg**
(AUTOMATIC) opensource update

### CODE REVIEW

**Comment by rsolomakhin** [comment]
Thank you for the update! Is there a plan to help fix the Travis CI build? The open-source community relies on that for making sure the code is correct.

**Comment by sebsg** [comment]
It looks like it started failing a while back when INSTANTIATE_TEST_CASE_P was replaced by INSTANTIATE_TEST_SUITE_P so maybe a dependency to a newer version of googletest?

File: .travis.yml [hunk 1]
```diff
@@ -44,17 +29,9 @@ install:
...
-  - g++ -std=c++11 -isystem ${GTEST_DIR}/include -I${GTEST_DIR} -pthread -c ${GTEST_DIR}/src/gtest-all.cc
-  - ar -rv libgtest.a gtest-all.o
-  - cd ${GTEST_DIR}/make
-  - make
 script:
-  - cd ${TRAVIS_CI_PATH}/cpp
-  - env CXXFLAGS="-I$HOME/usr/local/include -I${GTEST_DIR}/include -I${GTEST_DIR} -I${GTEST_DIR}/make" LDFLAGS="-L$HOME/usr/local/lib -L$(pwd) -L${GTEST_DIR}/include -L${GTEST_DIR} -L${GTEST_DIR}/make" CPPFLAGS="-I${GTEST_DIR}/include -I${GTEST_DIR} -I${GTEST_DIR}/make" GYP_GENERATORS="ninja"  gyp --depth .
+  - cd cpp
+  - env CXXFLAGS="-I$HOME/usr/local/include" LDFLAGS="-L$HOME/usr/local/lib" GYP_GENERATORS="ninja" gyp --depth .
   - ninja -C out/Default
```
Comment (parastoog) [inline .travis.yml]: My investigation result: the only way to make the gtests work is to stop using ninja, bc it uses the older versions. As far as I searched/tested they don't work together.

Comment (sebsg) [reply .travis.yml]: @rsolomakhin WDYT? Also @parastoog do you know why we made the change to SUITE from CASE?

Comment (parastoog) [reply .travis.yml]: It's explained in this document:
https://docs.google.com/document/d/1bcccx4SlrBwZmlMXNhp_v8WUdl_ph3CSvM99VsPzTpc/edit

**Pull request merged**